src/brouwers/kitreviews/templates/kitreviews/review_form.html

```diff
@@ -1,57 +1,25 @@
-{% load i18n %}
+{% load i18n sniplates %}
 
-<div id="review_kit">
-    {% with reviewform.model_kit as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
+{% block form %}
 
-<div id="review_raw_text">
-    {% with reviewform.raw_text as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
+    <fieldset class="well">
+        {% load_widgets form='horizontal.form' %}
+        {% form_field form.model_kit %}
```

Comment:
Also, in it's current form this is generating 300+ queries since the display value for a kit does a look up. You can quickly resolve this by overriding the model form `__init__` by doing something like this:

```
def __init__(self, *args, **kwargs):
    super(ReviewForm, self).__init__(self, *args, **kwargs)
    self.fields['model_kit'].queryset = self.fields['model_kit'].queryset.select_related('brand')
```